samples/outlook/default.yaml

```diff
@@ -9,7 +9,7 @@ script:
         $("#run").click(run);
 
         function run() {
-            OfficeHelpers.UI.notify(`Hello ${Office.context.mailbox.userProfile.displayName}`);
+            console.log(`Hello ${Office.context.mailbox.userProfile.displayName}`);
```

Comment:
To be consistent with the other samples (and make deleting stuff easier), should we make this a two-liner?

```
const displayName = Office.context.mailbox.userProfile.displayName;
console.log(`Hello ${displayName}`);
```